<?xml version="1.0"?>
<launch>
  <arg name="simulated" default="false"/>
  <arg name="torso_vrep_port" default="19999"/>
  <arg name="ergo_vrep_port" default="19998"/>

  <include file="$(find poppy_ergo_jr_controllers)/launch/controllers.launch" >
    <arg name="simulated" value="$(arg simulated)"/>
    <arg name="vrep_port" value="$(arg ergo_vrep_port)"/>
    <arg name="output" value="log"/>
  </include>

  <include file="$(find poppy_torso_controllers)/launch/controllers.launch" >
    <arg name="simulated" value="$(arg simulated)"/>
    <arg name="vrep_port" value="$(arg torso_vrep_port)"/>
    <arg name="output" value="log"/>
  </include>
</launch>